iroha-cli/client.hpp

```diff
@@ -59,6 +62,9 @@ namespace iroha_cli {
         std::string tx_hash);
 
    private:
+    std::string server_ip;
+    int server_port;
```

Comment:
1. Both field seems only setted but unused
2. Can be const now (don't really sure that's it good semantically)

Reply:
They are used in copy-constructors and interactive cli to send to server.